Teleportation only works sometimes: Could not pass event PlayerInteractEvent to LapisPortals v1.3.14

### Issue:

Sometimes teleportation works without issues, and sometimes it fails.
### How to reproduce:

Not yet known, sometimes it works, sometimes it does not.
When it works, it works multiple times in a row without issues.
When it fails, it fails multiple times in a row.
Usually just waiting or restarting the server might change whether it fails or works.
### When it fails, it's like this:

You close the door behind you, and nothing happens in-game, but you get this terminal output:

```
[11:00:29 ERROR]: Could not pass event PlayerInteractEvent to LapisPortals v1.3.14
org.bukkit.event.EventException
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306) ~[spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:62) ~[spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:502) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:487) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at org.bukkit.craftbukkit.v1_10_R1.event.CraftEventFactory.callPlayerInteractEvent(CraftEventFactory.java:231) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.PlayerInteractManager.a(PlayerInteractManager.java:492) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.PlayerConnection.a(PlayerConnection.java:890) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.PacketPlayInUseItem.a(SourceFile:55) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.PacketPlayInUseItem.a(SourceFile:11) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.PlayerConnectionUtils$1.run(SourceFile:13) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_102]
        at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_102]
        at net.minecraft.server.v1_10_R1.SystemUtils.a(SourceFile:45) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.MinecraftServer.D(MinecraftServer.java:732) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.DedicatedServer.D(DedicatedServer.java:400) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.MinecraftServer.C(MinecraftServer.java:668) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at net.minecraft.server.v1_10_R1.MinecraftServer.run(MinecraftServer.java:567) [spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        at java.lang.Thread.run(Thread.java:745) [?:1.8.0_102]
Caused by: java.lang.IndexOutOfBoundsException: Index: 0, Size: 0
        at java.util.ArrayList.rangeCheck(ArrayList.java:653) ~[?:1.8.0_102]
        at java.util.ArrayList.get(ArrayList.java:429) ~[?:1.8.0_102]
        at java.util.Collections$UnmodifiableList.get(Collections.java:1309) ~[?:1.8.0_102]
        at me.fetusdip.LapisPortals.PlayerListener.onPlayerInteract(PlayerListener.java:134) ~[?:?]
        at sun.reflect.GeneratedMethodAccessor193.invoke(Unknown Source) ~[?:?]
        at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43) ~[?:1.8.0_102]
        at java.lang.reflect.Method.invoke(Method.java:498) ~[?:1.8.0_102]
        at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:302) ~[spigot-1.10.2.jar:git-Spigot-f9671b8-48b88ed]
        ... 17 more
```

This error occurs using Spigot 1.10.2 and other versions after Bukkit 1.7.10 have not been tested.
If I remember correctly, this plugin worked fine with Bukkit 1.7.10 but it's been a while.

--- a/src/me/fetusdip/LapisPortals/PlayerListener.java
+++ b/src/me/fetusdip/LapisPortals/PlayerListener.java
@@ -1,5 +1,6 @@
 package me.fetusdip.LapisPortals;
 
+import java.util.List;
 import java.util.logging.Logger;
 
 import org.bukkit.ChatColor;
@@ -126,12 +127,11 @@ public class PlayerListener implements Listener {
 						}
 					}
 					if (toLoc != null) {
-						if ((event.getPlayer().hasMetadata("lpLastTele"))
+						List<MetadataValue> md = event.getPlayer().getMetadata("lpLastTele");
+						if ((md.size() >= 1)
 								&& (!VaultHook.hasPermission(event.getPlayer(),
 										VaultHook.Perm.NO_DELAY))) {
-							long dt = (System.currentTimeMillis() - ((MetadataValue) event
-									.getPlayer().getMetadata("lpLastTele")
-									.get(0)).asLong()) / 1000L;
+							long dt = (System.currentTimeMillis() - md.get(0).asLong()) / 1000L;
 							double delayRequired = this.plugin.getConfig()
 									.getDouble("TeleDelay");
 							if (dt < delayRequired) {

